docs/research/offline-rustsec.md

```
@@ -0,0 +1,308 @@
+|--------|-------|
+| Markdown files walked | 1167 |
+| Advisories (leading toml fence) | 1161 |
+| Parsed OK | 1161 |
+| Parse errors | 0 |
+| Distinct affected packages | 878 |
+| id = RUSTSEC-* | 1141 |
+| id = CVE-* (the `rust/` toolchain and std advisories) | 20 |
+| Withdrawn (`withdrawn` field set) | 13 |
+| Informational (`informational` field set) | 452 |
+| Advisories with at least one alias | 783 |
+| Alias occurrences CVE-* / GHSA-* / other | 583 / 904 / 6 |
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)